source/sass/31-molecules/map/_leaflet.scss

```diff
@@ -0,0 +1,151 @@
+$color-white: color('white');
+$color-gray: color('dark-gray');
+$color-dark-blue: color('blue', -1);
+$color-light-blue: color('blue', -3);
```

Comment:
Can you also use the theme() mixin as you did in version 6.x?
Look at the color themes in the styleguide 5.x to see which color variables are available. This can be different from styleguide 6.x.
These variables can be removed then.